app/main/views/briefs.py

```diff
@@ -106,10 +106,28 @@ def get_brief(brief_id):
     brief = Brief.query.filter(
         Brief.id == brief_id
     ).first_or_404()
-
     return jsonify(briefs=brief.serialize(with_users=True))
 
 
+@main.route('/briefs/teammembers/<string:email_domain>', methods=['GET'])
+def get_team_briefs(email_domain):
+    query = db.session.execute("""
+            select brief_ids from vuser_users_with_briefs
+            where email_domain = :domain
+        """, {'domain': email_domain})
+
+    user_brief_lists = list()
+    for i in [_ for _ in query if _[0][0] is not None]:
```

Comment:
empty lists returned from api as [None] representing domain users with no briefs. I removed it and Brief.query(.. handled them fine so might remove this loop